server/index.js

```diff
@@ -17,21 +17,12 @@ app.get('/prices/:ticker/:date', async (request, response) => {
     const apiUrl = `https://api.tiingo.com/tiingo/daily/${ticker}/prices?startDate=${apiDate}&endDate=${apiDate}&token=${tiingo_api_key}`;
     const fetchResponse = await fetch(apiUrl);
```

Comment:
I've used this; don't know if it's better or worst (and might even be wrong):
```
fetch(apiURL)
  .then(response => response.json())
  .then(callback)  // not sure what your callback is here
  .catch(err => { console.log(err); };  // TODO: maybe this goes before the callback?
```
I haven't found a definitive answer for how best to do this kind of thing. :(

Reply:
async await is intended to replace .then in most cases

Reply:
https://javascript.info/async-await